src/main/kotlin/com/sourcegraph/cody/autocomplete/CodyAutocompleteManager.kt

```diff
@@ -415,32 +415,22 @@ class CodyAutocompleteManager {
             .removeSuffix(originalTextAfterCursor)
     if (completionText.trim().isBlank()) return
 
+    // Determine if startInline or not
     val lineBreaks = listOf("\r\n", "\n", "\r")
     val startsInline = lineBreaks.none { separator -> completionText.startsWith(separator) }
 
-    var inlay: Inlay<*>? = null
-    if (startsInline) {
-      val renderer =
-          CodyAutocompleteSingleLineRenderer(
-              completionText.lines().first(), items, editor, AutocompleteRendererType.INLINE)
-      inlay =
-          inlayModel.addInlineElement(cursorOffset, /* relatesToPrecedingText = */ true, renderer)
-    }
-    val lines = completionText.lines()
-    if (lines.size > 1) {
-      val text =
-          (if (startsInline) lines.drop(1) else lines).dropWhile { it.isBlank() }.joinToString("\n")
-      val renderer = CodyAutocompleteBlockElementRenderer(text, items, editor)
-      val inlay2 =
-          inlayModel.addBlockElement(
-              /* offset = */ cursorOffset,
-              /* relatesToPrecedingText = */ true,
-              /* showAbove = */ false,
-              /* priority = */ Int.MAX_VALUE,
-              /* renderer = */ renderer)
-      if (inlay == null) {
-        inlay = inlay2
-      }
+    val rendererType = if (startsInline) AutocompleteRendererType.INLINE else AutocompleteRendererType.BLOCK
+    val renderer = CodyAutocompleteRenderer(completionText, items, editor, rendererType)
+
+    val inlay = if (startsInline) {
```

Comment:
Should we move this to the CodyAutocompleteRenderer?

Or at the very least, we could look at something like `renderer.type` instead of the `startsInline` we're hanging on to.